github-upload-secrets

```diff
@@ -122,7 +122,7 @@ def main():
     api = task.github.GitHub()
 
     parser = argparse.ArgumentParser(description='Upload encrypted action secrets to GitHub')
-    parser.add_argument('--repository', '-r', '--receiver', metavar="OWNER/REPO",
+    parser.add_argument('--repository', '-r', '--receiver', metavar="OWNER/REPO", required=True,
```

Comment:
Can you please give a rationale in the commit message? This seemed quite useful to me, to avoid duplicating the project name in the deploy-keys.sh script.

Reply:
We don't use this feature, though.  Take a look at our existing `deploy-keys.sh` files:
 - https://github.com/cockpit-project/cockpit-machines/blob/main/.github/deploy-keys.sh
 - https://github.com/cockpit-project/cockpit/blob/main/.github/deploy-keys.sh

both of those are doing, in one way or another, something like: `--receiver "${ORG}/${THIS}"`

Reply:
Ack. I suppose that was still from the days when we called this manually.